src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/UpdateDescriptorsValidation.feature

```diff
@@ -215,14 +209,8 @@ Feature: Update a Descriptor
                   """
                   {
                     "validationErrors": {
-                      "$.codeValue": [
-                        "codeValue cannot contain leading or trailing spaces."
-                      ],
                       "$.namespace": [
                         "namespace cannot contain leading or trailing spaces."
-                      ],
-                      "$.shortDescription": [
-                        "shortDescription cannot contain leading or trailing spaces."
```

Comment:
This is good, but let's also add a test like your new create scenario 18, that updates successfully with whitespace and the whitespace is removed.